### PR TITLE
[ethercats-founder-series] added ethercats nft strategy

### PR DESCRIPTION
**Pull request by naomsa**

### CODE REVIEW

**Comment by ChaituVR** [comment]
@naomsa#5548 Also there is a heap memory error on GitHub action and test cases fail

image

**Comment by ChaituVR** [comment]
I think we should try the solution with subgraph  https://thegraph.com/hosted-service/subgraph/alexvorobiov/eip1155subgraph

**Comment by woodydeck** [comment]
> I think we should try the solution with subgraph https://thegraph.com/hosted-service/subgraph/alexvorobiov/eip1155subgraph

This subgraph reports ownership incorrectly for the collection. This subgraph probably shouldn't be used for anything without a patch and full audit. It's not really an edge case either, it will affect all collections using Chainlink VRF I believe to start with.

Is there another subgraph we can validate against?

File: src/strategies/ethercats-founder-series/index.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import { subgraphRequest } from '../../utils';
+
+export const author = 'naomsa';
```
Comment (ChaituVR) [inline src/strategies/ethercats-founder-series/index.ts:3]:
```suggestion
export const author = 'woodydeck';
```

File: src/strategies/ethercats-founder-series/examples.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "name": "Example query",
+    "strategy": {
+      "name": "ethercats-founder-series",
+      "params": {
+        "symbol": "Ether Cat"
```
Comment (ChaituVR) [inline src/strategies/ethercats-founder-series/examples.json:7]:
```suggestion
        "symbol": "EtherCat"
```

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**